packages/api-gateway/src/drivers/http/adapters/places/handlers.js

```diff
@@ -14,6 +14,25 @@ async function postPlace(req, reply) {
     .send({ msg: `${place_name} has been saved correctly.` });
 }
 
+async function getPlaces(req, reply) {
+  req.log.info('[http-server]: reading places');
+
+  let places;
+  try {
+    places = await this.placesService.getPlaces();
+  } catch (error) {
+    return reply.code(500).send({
+      message: 'Could not get places',
+    });
```

Comment:
@Alejandrocuartas we shouldn't return 500 if we are handling this error, so I would recommend getting the error precedence, and based on that kind of error we can return a proper response. Let's bear in mind that 500 are for uncaught errors.

Reply:
Done.